src/settingsholder.cpp

```diff
@@ -177,6 +183,8 @@ void SettingsHolder::clear() {
   m_settings.remove(SETTINGS_SURVEYS);
   m_settings.remove(SETTINGS_IAPPRODUCTS);
   m_settings.remove(SETTINGS_POSTAUTHENTICATIONSHOWN);
+  m_settings.remove(SETTINGS_NEWFEATURESSEEN);
+  m_settings.remove(SETTINGS_FEATURESTOURSHOWN);
```

Comment:
This means that after a logout, the user will see all the new features again. Is it OK?

Reply:
Thank you for catching this. No it’s not OK and I’ll remove this.